packages/web-components/src/accordion-item/accordion-item.ts

```diff
@@ -0,0 +1,40 @@
+import { attr } from '@microsoft/fast-element';
+import { FASTAccordionItem } from '@microsoft/fast-foundation';
+import { AccordionItemExpandIconPosition, AccordionItemSize } from './accordion-item.options.js';
+
+/**
+ * @internal
+ */
+export class AccordionItem extends FASTAccordionItem {
+  /**
+   * Defines accordion header size.
+   *
+   * @public
+   * @default 'medium'
+   * @remarks
+   * HTML Attribute: appearance
+   */
+  @attr({ attribute: 'size' })
+  public size: AccordionItemSize;
+
+  /**
+   * Sets the width of the focus state.
+   *
+   * @public
+   * @remarks
+   * HTML Attribute: block
+   */
+  @attr({ mode: 'boolean' })
+  public block: boolean = true;
```

Comment:
How is a boolean attribute which defaults to `true` supposed to work? How can I set it to `false`?
(even if that had worked technically, true by default is an antipattern)

Reply:
Good catch on this - boolean should always default to false. It doesn't work :)